mirrord-layer/src/socket.rs

```diff
@@ -125,8 +125,10 @@ pub struct UserSocket {
 }
 
 #[inline]
-const fn is_ignored_port(port: Port) -> bool {
-    port == 0 || (port > 50000 && port < 60000)
+fn is_ignored_port(addr: SocketAddr) -> bool {
+    let (ip, port) = (addr.ip(), addr.port());
+    let ignored_ip = ip == IpAddr::V4(Ipv4Addr::LOCALHOST) || ip == IpAddr::V6(Ipv6Addr::LOCALHOST);
+    port == 0 || ignored_ip && (port > 50000 && port < 60000)
```

Comment:
Aren't these ports the VSCode debugger ports, and thus should be passed from the VS Code extension?